### PR TITLE
 Notes about FSevent error and jest

### PR DESCRIPTION
**Pull request by peterbe**
We discussed this in channel and I fear it's only matter of time till someone else falls for it. Better written down.

### CODE REVIEW

File: docs/dev.rst [hunk 1]
```diff
@@ -74,3 +74,36 @@ When working on an individual app, there is a need to stub out certain values/pr
...
+          ^
+    
+    Error: EMFILE: too many open files, watch
+        at FSEvent.FSWatcher._handle.onchange (fs.js:1372:28)
+    Emitted 'error' event at:
+        at FSEvent.FSWatcher._handle.onchange (fs.js:1378:12)
+    error Command failed with exit code 1.
+    info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
+    
+A probably cause is that you don't have ``watchman`` installed. For example, on macOS you can fix this by installing...:
```
Comment (magopian) [inline docs/dev.rst]: typo: `probable`

**Comment by peterbe** [comment]
Oops! I only just now realized that when you edit in GitHub like this, it creates a branch in *this* repo and now in my fork. Sorry.

**Comment by peterbe** [comment]
bors=r+

(is that how you bors?)

**Comment by peterbe** [comment]
bors r= @magopian

**Pull request merged**